usr/src/uts/common/fs/zfs/zil.c

```diff
@@ -1057,12 +1061,13 @@ zil_lwb_write_start(zilog_t *zilog, lwb_t *lwb)
 static lwb_t *
 zil_lwb_commit(zilog_t *zilog, itx_t *itx, lwb_t *lwb)
 {
-	lr_t *lrc = &itx->itx_lr; /* common log record */
-	lr_write_t *lrw = (lr_write_t *)lrc;
+	lr_t *lrcb, *lrc = &itx->itx_lr; /* common log record */
```

Comment:
One set of variables there points to record in original itx buffer, while another set points to record copy in lwb buffer.  New code logic requires ability to access/modify both of them when splitting single itx record into multiple lwb records.